src/application/shell.ts

```diff
@@ -494,6 +533,22 @@ namespace ApplicationShell {
      */
     rank?: number;
   }
+
+  /**
+   * An arguments object for the changed signals.
+   */
+  export
+  interface IChangedArgs {
```

Comment:
this could be a type alias `type IChangedArgs = FocusTracker.IChangedArgs<Widget>`